kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/kms/ExperimentalKmsConfig.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.resources.kms;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record ExperimentalKmsConfig(
+                                    @JsonProperty long resolvedAliasExpireAfterWriteSeconds,
```

Comment:
Make these Long so that you can choose to set a value only for the ones you really want to override.